build.sh

```diff
@@ -1,7 +1,7 @@
 cd build-babel
 jspm install --quick --log err
-jspm build systemjs-babel.js ../systemjs-babel-browser.js --format amd --skip-source-maps --minify --log ok
-jspm build systemjs-babel.js ../systemjs-babel-node.js --format amd --node --skip-source-maps --log ok
+jspm build systemjs-babel.js ../systemjs-babel-browser.js --format cjs --skip-source-maps --minify --log ok
+jspm build systemjs-babel.js ../systemjs-babel-node.js --format cjs --node --skip-source-maps --log ok
```

Comment:
What was the reason for the change to `cjs` here? AMD as a format runs without source regexs so is in theory faster for larger files.